tests/e2e-instrumentation/instrumentation-dotnet-multiannotation/00-install-collector.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-other-dotnet-ns
+  annotations:
+    instrumentation.opentelemetry.io/inject-sdk: "true"
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: sidecar
+  namespace: my-other-dotnet-ns
+spec:
+  config: |
```

Comment:
let's use the new collector version here :)

Reply:
Just fixed.